source/accessories/uinverter/hardware-design.md

```diff
@@ -185,6 +185,23 @@ V_\text{amp} &= (\text{amplifier gain}) \times (\text{current} \times R_\text{se
 When the DC supply voltage exceeds 12V, the phase current increases significantly and may push the current sense amplifier in saturation.
 To aviod any damage to components on the AMDC board, the DC supply voltage must be limited to no more than 12V.  
 
+**DC link voltage sensing filter:**
```

Comment:
Change to be third level header ("###") and capitalize words to match page formatting

Reply:
I made the changes